static/js/getCsrfToken.js

```diff
@@ -1,16 +1,12 @@
-export default function getCsrfToken(options) {
-    return getCookie('csrftoken');
-}
-
-function getCookie(name) {
+export default function getCsrfToken() {
     let cookieValue = null;
     if (document.cookie && document.cookie !== '') {
         const cookies = document.cookie.split(';');
         for (let i = 0; i < cookies.length; i++) {
             const cookie = cookies[i].trim();
             // Does this cookie string begin with the name we want?
-            if (cookie.substring(0, name.length + 1) === (name + '=')) {
-                cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+            if (cookie.substring(0, 10) == ('csrftoken' + '=')) {
```

Comment:
can you explain what bug this fixes? How did it used to fail?

Reply:
Not a bug per say, but since we know both the name & length of the token, why not state it?

Reply:
because this code was a generic re-usable getCookie function ( recommended from django docs )

Reply:
oooooh. Alrighty then, will revert that.